docs/yamlpages/cookbook.rst

```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
+of these options are good stock settings to remember.
+
+In this example:
+
+* Targeting folders for input files
+* Configuring Output files
+* Setting good stock options for implicit simulations
+* Setting up simple protein/ligand binding simulation
+* NVT ensemble
+
+.. code-block:: yaml
+
+  options:
+    verbose: yes
+    setup_dir: setup
+    output_dir: output
+    experiments_dir: experiments
+    randomize_ligand: yes
+    restraint_type: flat-bottom
+    minimize: yes
+    number_of_iterations: 2000
+    temperature: 300*kelvin
+  
+  molecules:
+    T4_lysozyme:
+      filepath: T4.pdb
+    p-xylene:
+      smiles: CC1=CC=C(C=C1)C
+      antechamber:
+        charge_method: bcc
+    
+  solvents:
+    GBSA:
+      nonbonded_method: NoCutoff
+      implicit_solvent: OBC2
+      solvent_dielectric: 78.5
+
+  systems:
+    T4-xylene-complex:
+      receptor: T4_lysozyme
+      ligand: p-xylene
+      solvent: GBSA
+      pack: yes
+      leap:
+        parameters: [leaprc.ff14SB, leaprc.gaff]
+
+  protocols:
+    absolute-binding:
+      phases:
+        complex:
+          alchemical_path:
+            lambda_electrostatics: [1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.55, 0.50, 0.45, 0.40, 0.35, 0.30, 0.25, 0.20, 0.15, 0.10, 0.05, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
+            lambda_sterics:        [1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.55, 0.50, 0.45, 0.40, 0.35, 0.30, 0.25, 0.20, 0.15, 0.10, 0.00]
+        solvent:
+          alchemical_path:
+            lambda_electrostatics: [1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.55, 0.50, 0.45, 0.40, 0.35, 0.30, 0.25, 0.20, 0.15, 0.10, 0.05, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
+            lambda_sterics:        [1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.55, 0.50, 0.45, 0.40, 0.35, 0.30, 0.25, 0.20, 0.15, 0.10, 0.00]
+
+  experiemnts:
+    system: T4-xylene-complex
+    protocol: absolute-binding
+                
+After this is all setup, simply run: ``yank script --yaml={ThisScriptName.yaml}`` and YANK will take care of the rest for you.
+
+
+----
+
+.. _yaml_ex_explicit:
+
+Example: Absolute Binding free energy in explicit solvent
+=========================================================
+
+This example takes the same para-xylene binding to T4-Lysozyme system as before, but now uses an explicit solvent setup, 
+minimal options, and automatic water addition (TIP3P).
+This example also sets up MPI so you can run on a cluster; assumes 4 nodes are available
```

Comment:
The YAML script does not have control on whether yank is run through `mpirun` or simply through `python`. It only has automatic detection on whether it's running with MPI or not.